packages/fxa-payments-server/.storybook/components/MockApp.tsx

```diff
@@ -92,11 +92,7 @@ export const MockApp = ({
     <AppContext.Provider
       value={{ ...appContextValue, navigatorLanguages: languages }}
     >
-      <AppLocalizationProvider
-        baseDir="./locales"
-        userLocales={languages}
-        bundles={['payments', 'react']}
-      >
+      <AppLocalizationProvider baseDir="./locales" userLocales={languages}>
```

Comment:
Was going to ask if missing `bundles` here causes any issues in Storybook but looks like l10n didn't work in payments anyway (not surprised since @vpomerleau recently fixed that for settings).

Reply:
It'll default to the main bundle if nothing is provided.

Reply:
I confirm, l10n for payments storybook previously worked.